includes/admin/views/html-admin-setup-step-1.php

```diff
@@ -11,22 +11,34 @@
 ?>
 <h3><?php esc_html_e( 'Welcome to the Setup Wizard!', 'wp-job-manager' ); ?></h3>
 
+<script type="text/javascript">
```

Comment:
I don't understand why we don't do this in PHP directly. Changed that in 00ba0f023fd6ecbedda56cb3d2e2152514452246

@mikeyarce let me know if there is something that I'm missing here.